packages/cavez-of-phear/build.sh

```diff
@@ -1,9 +1,9 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/haakonnessjoen/Cavez-of-Phear-packaging
+TERMUX_PKG_HOMEPAGE=https://github.com/AMDmi3/cavezofphear
 TERMUX_PKG_DESCRIPTION="A Boulder Dash like game for consoles/terminals"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Henrik Grimler @Grimler91"
```

Comment:
If you are maintaining the upstream package now, consider taking some credits here too :)

Reply:
I'm ok with taking maintainership here as long as current maintainer @Grimler91 is ok with that.

Reply:
Yeah, absolutely :+1: